sortinghat/core/schema.py

```diff
@@ -826,10 +827,8 @@ def resolve_organizations(self, info, filters=None,
             query = query.filter(name=filters['name'])
         if filters and 'term' in filters:
             search_term = filters['term']
-            query = query.filter(Q(name__icontains=search_term) |
-                                 Q(name__in=Subquery(Domain.objects
-                                                    .filter(domain__icontains=search_term)
-                                                    .values_list('organization__name'))))
+            query = query.filter(Q(name__icontains=search_term) | Q(name__in=Subquery(
```

Comment:
I rather keep this as in the original file. The query is easier to ready. Is there anyway to avoid this problem?

Reply:
Yes, we can do it. The flake8 documentation has a mention about it.
https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors